sdk/identity/Azure.Identity/src/Credentials/OnBehalfOfCredential.cs

```diff
@@ -86,6 +86,30 @@ public OnBehalfOfCredential(string tenantId, string clientId, X509Certificate2 c
             : this(tenantId, clientId, clientSecret, userAssertion, options, null, null)
         { }
 
+        /// <summary>
+        /// Creates an instance of the <see cref="OnBehalfOfCredential"/> with the details needed to authenticate against Microsoft Entra ID with the specified certificate.
+        /// </summary>
+        /// <param name="tenantId">The Microsoft Entra tenant (directory) ID of the service principal.</param>
+        /// <param name="clientId">The client (application) ID of the service principal</param>
+        /// <param name="clientAssertionCallback">An asynchronous callback returning a valid client assertion used to authenticate the service principal.</param>
+        /// <param name="userAssertion">The access token that will be used by <see cref="OnBehalfOfCredential"/> as the user assertion when requesting On-Behalf-Of tokens.</param>
+        /// <param name="options">Options that allow to configure the management of the requests sent to Microsoft Entra ID.</param>
+        public OnBehalfOfCredential(string tenantId, string clientId, Func<CancellationToken, Task<string>> clientAssertionCallback, string userAssertion, OnBehalfOfCredentialOptions options = null)
+            : this(tenantId, clientId, null, clientAssertionCallback, userAssertion, null, null, null)
```

Comment:
`options` param is not being passed in here. Maybe it should.

Reply:
good catch!